src/middlewared/middlewared/plugins/failover.py

```diff
@@ -435,6 +435,22 @@ async def get_disks_local(self):
         except Exception:
             self.logger.error('Unhandled exception in get_disks_local', exc_info=True)
 
+    @private
+    def mismatch_nics(self):
```

Comment:
We're going to have to add an alert for this new check. Furthermore, this method doesn't really match what `failover.mismatch_disks` does.

The mismatch_disks method actually tracks what's missing on the local node and what's missing on the remote node. We need to do the same here with this method.

Finally,  you're going to need to add an alert as well.